Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/wordpress-mobile/release-toolkit
-  revision: 6c1fa45f3beb216f4f5d03346f2981f1024b799e
-  tag: 0.9.6
+  revision: 4c2b3b0403979b4d6c8485bbd54f27031e8f795e
+  branch: traslator-review-build
```

Comment:
Leaving this comment here just so we remember to point back to a released version before merging the PR 😉